src/test/resources/edu/ie3/datamodel/io/source/sql/_weather/icon/weather.sql

```diff
@@ -6,8 +6,8 @@ CREATE TABLE public.weather
     aswdifd_s double precision,
     aswdifu_s double precision,
     aswdir_s double precision,
-    t_2m double precision,
```

Comment:
this is the air temperature (measured at 2m high). This needs to stay in! (and maybe this causes your failing tests?)